static/components/MobileLayout.js

```diff
@@ -42,38 +42,87 @@ const MobileLayout = () => {
     showComparisonModal,
```

Comment:
definitely wanna move away from this whole mobile layout view on a final cleanup